internal/commands/user/disable_test.go

```diff
@@ -0,0 +1,270 @@
+						return nil, errors.New("something bad happened")
+					}
+					return realmClient
+				},
+				expectedErr: errors.New("something bad happened"),
+			},
+			{
+				description: "when finding the users fails",
+				setupClient: func() realm.Client {
+					realmClient := mock.RealmClient{}
+					realmClient.FindAppsFn = func(filter realm.AppFilter) ([]realm.App, error) {
+						return []realm.App{testApp}, nil
+					}
+					realmClient.FindUsersFn = func(groupID, appID string, filter realm.UserFilter) ([]realm.User, error) {
+						return nil, errors.New("something bad happened")
+					}
+					return realmClient
+				},
+				expectedErr: errors.New("something bad happened"),
+			},
+		} {
+			t.Run(tc.description, func(t *testing.T) {
+				realmClient := tc.setupClient()
+				cmd := &CommandDisable{
+					realmClient: realmClient,
+				}
+				err := cmd.Handler(nil, nil)
+
+				assert.Equal(t, tc.expectedErr, err)
+			})
+		}
+	})
+}
+
+func TestUserDisableFeedback(t *testing.T) {
+	testUsers := []realm.User{
+		{
+			ID:         "user-1",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Type:       "type-1",
+			Data:       map[string]interface{}{"email": "user-1@test.com"},
+		},
+		{
+			ID:         "user-2",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Type:       "type-2",
+			Data:       map[string]interface{}{"email": "user-2@test.com"},
+		},
+		{
+			ID:         "user-3",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Type:       "type-1",
+			Data:       map[string]interface{}{"email": "user-3@test.com"},
+		},
+		{
+			ID:         "user-4",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAPIKey}},
+			Type:       "type-1",
+			Data:       map[string]interface{}{"name": "name-4"},
+		},
+		{
+			ID:         "user-5",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeCustomToken}},
+			Type:       "type-3",
+		},
+	}
+	for _, tc := range []struct {
+		description     string
+		outputs         []userOutput
+		expectedContent string
+	}{
+		{
+			description:     "should show no users to disable",
+			outputs:         []userOutput{},
+			expectedContent: "01:23:45 UTC INFO  No users to disable\n",
+		},
+		{
+			description: "should show 1 failed user",
+			outputs: []userOutput{
+				{user: testUsers[0], err: errors.New("client error")},
+			},
+			expectedContent: strings.Join(
+				[]string{
+					"01:23:45 UTC INFO  Provider type: User/Password",
+					"  Email            ID      Type    Enabled  Details     ",
+					"  ---------------  ------  ------  -------  ------------",
+					"  user-1@test.com  user-1  type-1  true     client error",
+					"",
+				},
+				"\n",
+			),
+		},
+		{
+			description: "should show failures to disable 2 users amongst successful results across different auth provider types",
+			outputs: []userOutput{
```

Comment:
[meganit] use the new type here: `userOutputs`, and reduce some text-noise with removing the `user:` and `err:` field declarations below this